documentation/asciidoc/computers/raspberry-pi/boot-eeprom.adoc

```diff
@@ -13,10 +19,13 @@ If an error occurs during boot, then an xref:configuration.adoc#led-warning-flas
 
 There are multiple ways to update the bootloader of your Raspberry Pi.
 
-==== Raspberry Pi 5, Raspberry Pi 4 and Raspberry Pi 400
+==== Raspberry Pi 5, Compute Module 5, Raspberry Pi 500, Raspberry Pi 4 and Raspberry Pi 400
```

Comment:
Bit of a nitpick, but when listing a collection of different models, would it make sense to always list them in the same order for consistency?
I think my preference would be to list flagship models before compute modules, and within each category list things in the order that they're released? So "Raspberry Pi 4, Raspberry Pi 400, Raspberry Pi 5, Raspberry Pi 500 and Compute Module 5", but YMMV.

Reply:
```suggestion
==== Flagship models since 4; Compute Modules since 5; Keyboard models since 400
```
Even more generic, forward-compatible, and terse.